server/controllers/new.js

```diff
@@ -1,161 +0,0 @@
-import db from '../models';
-
-const { Recipe, Review } = db;
-
-export default {
-  create(req, res) {
-    const { userDetail } = req.decoded;
-    Recipe.findOne(({
-      where: {
-        $or: [
-          {
-            recipeName: req.body.recipeName
-          },
-          { recipeId: req.params.recipeId }
-        ]
-      },
-    }));
-    if (!userDetail) {
-      return res.status(400)
-        .json({ error: 'user not found' });
-    }
-    return Recipe
-      .create({
-        recipeName: req.body.recipeName,
-        description: req.body.description,
-        ingredients: req.body.ingredients,
-        userId: userDetail.id
-      })
-      .then(newRecipe => res.status(201).json({
-        recipeName: newRecipe.recipeName,
-        description: newRecipe.description,
-        ingredients: newRecipe.ingredients,
-        recipeId: newRecipe.id,
-        userId: newRecipe.userId
-      }))
-      .catch((error) => {
-        res.status(400).json({ error: error.message });
-      })
-      .catch(() => {
-        res.status(500).json({ error: 'oops something went wrong' });
-      });
-  },
-
-  listAllRecipes(req, res) {
-    const limitValue = req.query.limit || 5;
-    const pageValue = (req.query.page - 1) || 0;
-    const sort = req.query.sort === 'upvotes' ||
-      req.query.sort === 'downvotes' ?
-      req.query.sort : 'upvotes';
-    const order = req.query.order === 'des' ?
-      'DESC' : 'DESC';
-    Recipe
-      .findAndCountAll({
-        order: [
-          [sort, order]
-        ],
-        limit: limitValue,
-        offset: pageValue * limitValue
-      })
-      .then(recipeList => res.status(200).send({
-        page: (pageValue + 1),
-        totalCount: recipeList.count,
-        pageCount: Math.ceil(recipeList.count / limitValue),
-        pageSize: parseInt(recipeList.rows.length, 10),
-        allRecipes: recipeList.rows
-      }))
-      .catch(() => res.status(422).send('Recipe Not Created yet'));
-  },
-
-  update(req, res) {
-    const { userDetail } = req.decoded;
-    return Recipe
-      .find({
-        where: {
-          id: req.params.recipeId,
-        },
-      }).then((Recipefound) => {
```

Comment:
Shouldn't `RecipeFound` should be `recipeFound` to keep consistent camel-case naming convention for variables?

Reply:
Thanks. Just realized this file is deleted.